components/Tabs/Tabs.js

```diff
@@ -2,22 +2,38 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+      this.element=element
     
     // Get the custom data attribute on the Link
-    // this.data;
+      this.data=this.element.dataset.tab;
     
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
+     this.itemElement=document.querySelector(`.tabs-item[data-tab='${this.data}']`);
     
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
+    this.tabItem= new TabItem(this.itemElement);
     
     // Add a click event listener on this instance, calling the select method on click
+    //this.element.addEventListener("click", this.select.bind(this));
+    this.element.addEventListener('click',(event=>{
+     this.select(event)
+    }))
+  
 
   };
 
   select() {
+    let links=document.querySelectorAll('.tabs-link');
+    console.log(links);
+
+    Array.from(links).forEach(function(element){
```

Comment:
No need for Array.from here, Node Lists have their own forEach method you can use